aayushi/src/components/BackGround.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+
+export const BackGround = ({ type }) => {
+  return (
+    <div className="">
+      <div className="w-full h-screen">
+        <div className="bg-[url('src/assets/bg.jpg')] bg-center grid h-screen bg-cover bg-auth">
```

Comment:
Import image as import BGImage from "path" and then use it as <img src={BGImage} .../>